docs/source/index.rst

```diff
@@ -20,7 +20,6 @@
    contain the root `toctree` directive.
 
 .. |spec| replace:: |short_name| specification
-.. _spec: https://spec.oneapi.com/versions/latest/elements/oneDAL/source/index.html
```

Comment:
it should be fixed and not removed - 

https://oneapi-spec.uxlfoundation.org/specifications/oneapi/v1.3-rev-1/elements/onedal/source/#